share/OpenMS/CHEMISTRY/Enzymes.xml

```diff
@@ -92,7 +92,14 @@
       <ITEM name="COMETid" value="3" type="int" />
     </NODE>
     <NODE name="Lys-N">
-    <ITEM name="COMETid" value="4" type="int" />
+      <ITEM name="Name" value="Lys-N" type="string" />
+      <ITEM name="RegEx" value="(?=K)" type="string" />
+      <ITEM name="RegExDescription" value="Lys-N cuts before K" type="string" />
+      <ITEM name="NTermGain" value="OH" type="string" />
+      <ITEM name="CTermGain" value="H" type="string" />
+      <ITEM name="PSIid" value="10013??" type="string" />
```

Comment:
this might result in an error. probably better to keep this empy or propose the enzyme to the PSI-MS